Tangem/UIComponents/BalanceWithButtonsView/BalanceWithButtonsView.swift

```diff
@@ -32,6 +37,19 @@ struct BalanceWithButtonsView: View {
         .background(Colors.Background.primary)
         .cornerRadiusContinuous(14)
     }
+
+    @ViewBuilder
+    private var balancePicker: some View {
+        if let balanceTypeValues = viewModel.balanceTypeValues {
+            SegmentedPicker(
+                selectedOption: $viewModel.selectedBalanceType,
+                options: balanceTypeValues,
+                shouldStretchToFill: false
+            ) { $0.title }
+        } else {
+            EmptyView()
+        }
```

Comment:
nit: просто на будущее )
```suggestion
        }
```